src/index.html

```diff
@@ -1,17 +1,201 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">
+            <span>Surname: </span>
+            <input
+              id="surname"
+              name="surname"
+              type="text"
+              autocomplete="off"
+              required>
```

Comment:
```suggestion
            <input
              id="surname"
              name="surname"
              type="text"
              autocomplete="off"
              required
            >
```
Форматування. Закриваюча дужка має бути на тому ж рівні відступів, що і відкриваюча